code/infrastructure-api/package.json

```diff
@@ -25,10 +25,11 @@
     "winston": "2.3.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.5.5",
```

Comment:
I don't think you want these changes in this PR as there is another open PR to update all infrastructure-api dependencies (and the build is failing for this PR)

Reply:
Will revoke these changes